packages/tocco-ui/src/Menu/Menu.js

```diff
@@ -0,0 +1,43 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+const MenuStyles = styled.ul`
+  && {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    /* width: fit-content; best option for dense mode but not supported for IE */
+  }
+`
+
+/**
+ * Structure <Button> and <ButtonLink> hierarchically as a Menu.
+ */
+
+class Menu extends React.Component {
+  childs // eslint-disable-next-line
+  = React.Children.map(this.props.children, child => {
+    return React.cloneElement(child, {look: this.props.look})
+  })
```

Comment:
move to `render()`, rename to `children` (see comment in Item.js)

Reply:
see 87d87670dbeb51c2609d4a43efc9e8e487a6c1ee